### DISCUSSION TITLE
How does the rendering in Skia work?

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#19038] [Q&A] [answered]
In my app that runs on Raspberry Pi CM4 on a 800x480 touchscreen my application is typically using around 40% CPU time when idle. There's really no processing done (listening on some sockets, etc), so without much investigation I think it's the Skia rendering that is using most of the CPU time. On Windows I don't see this (but my Windows PC is quite high performance). I'm using X11 and to my knowledge it's using the acceleration. The only thing that happens on the screen is a clock that updates (via binding) once per second.
My question is if this is normal or if something is wrong? Sometimes I need to do some low-latency processing (recording a network stream) where timing is critical, is it possible to slow down the Skia screen updates (if that's what's consuming the CPU time)? When recording I only updated a duration label every 500ms with an option to Abort/Save (buttons).

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. Let's track in https://github.com/unoplatform/uno/issues/19042
